app/views/users/_unfollow.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_for unfollow, html: {method: :delete}, remote: true do |f| %>
```

Comment:
Tại sao phải để method: :delete ở đây em?

Reply:
Cái này để gửi Delete request đến hàm destroy trong relationship ạ

Reply:
Cái này gửi delete request để hủy relationship

Reply:
do delete không thể gửi từ browser, nên ta sử dụng thông qua for mảo ajax